src/electron/release_linux_action.sh

```diff
@@ -30,5 +61,8 @@ electron-builder \
   --publish never \
   --config src/electron/electron-builder.json \
   --config.extraMetadata.version=$(scripts/semantic_version.sh -p linux) \
+  --config.generateUpdatesFilesForAllChannels=true \
```

Comment:
No, that's for forcing an update channel.  Otherwise electron-build defaults to detecting the update channel from the version string.  https://www.electron.build/generated/platformspecificbuildoptions ctrl-F `detectUpdateChannel`